HW_2/task_1.py

```diff
@@ -0,0 +1,21 @@
+# Задача 16: Требуется вычислить, сколько раз встречается некоторое число X в массиве A.
+# Пользователь в первой строке вводит натуральное число N – количество элементов в массиве.
+# В последующих  строках записаны N целых чисел Ai (можно использовать псевдогенерацию).
+# Последняя строка содержит число X.
+#
+# *Пример:*
+#
+# 5
+#     7 -2 3 5 1
+#     3
+#     -> 1
+
+N = int(input('Введите ко-во элементов: '))
+list_1 = list(range(1, N + 1))
+print(list_1)
+count = 0
+X = int(input('Введите число: '))
+for i in range(0, len(list_1)):
```

Comment:
Во-первых, ноль указывать не обязательно, если им начинается с нуля. Во-вторых, у вас уже есть параметр, который содержит длину списка.
```suggestion
for i in range(n):
```
И в третьих, можно проходиться напрямую по элементам списка, не используя функцию `range`